packages/sdk/src/middleware/middleware.js

```diff
@@ -70,6 +72,7 @@ export default function createSdkMiddleware({
           ...(action.payload.headers || {}),
           ...(shouldRenewToken ? { 'X-Force-Token': 'true' } : {}),
           'X-Project-Key': projectKey,
+          ...(teamId && { 'X-Team-Id': teamId }),
```

Comment:
👍 
yeah, as discussed we might have to add `store` in the future

Reply:
You mean as an extra HTTP header?

Reply:
Yes, so use case being: a user is member of team A and B wheere A has store access D, E and B has store access F, G while the user browsers the MC as team A _and_ only wants to see store E (not D). Let's see when we get there.

Reply:
Let's see if it has to be a header or just can be part of a query e.g. `inStore(key: "luxury-brand")`.